testutil/testutil.go

```diff
@@ -62,8 +64,12 @@ func mustCreateDescriptorProtoFromSource(filename string, source io.Reader, deps
 		fmt.Sprintf("--descriptor_set_out=%s", descSetF.Name()),
 	}
 
-	if len(deps) > 0 {
-		descSetIn := mustCreateDescSetFileFromFileDescriptorProtos(deps)
+	for _, p := range spec.AdditionalProtoPaths {
+		args = append(args, fmt.Sprintf("--proto_path=%s", p))
+	}
+
+	if len(spec.Deps) > 0 {
+		descSetIn := mustCreateDescSetFileFromFileDescriptorProtos(spec.Deps)
```

Comment:
I think we can remove `FromFileDescriptorProtos`, otherwise the function name is really, really long.

Reply:
Done.